voc/access_restriction.go

```diff
@@ -29,6 +31,6 @@ type AccessRestriction struct {
 	*Authorization
 }
 
-func (*AccessRestriction) Type() string {
+func (a *AccessRestriction) Type() string {
```

Comment:
If it is not too much work I would prefer leaving the receiver our in the Type() functions. I think we cannot ignore deep source issues just for specific files and I think the way without receiver is actually cleaner.

Reply:
True, this is the simplest solution :+1:  That's easy to change.